src/Tribe/Cache/Abstract_Cache.php

```diff
@@ -104,7 +125,8 @@ protected function fetch_posts_without_ticket_types( array $post_types = null )
 
 		$query = "SELECT DISTINCT(ID) FROM {$wpdb->posts}
 				WHERE post_type IN {$post_types}
-				AND post_status != 'auto-draft'";
+				AND post_status != 'auto-draft'
+				AND post_status != 'trash'";
```

Comment:
We should allow filtering of these post status and use `NOT IN` to allow easier filtering.